### PR TITLE
Tp #42 add comment on post

### PR DESCRIPTION
**Pull request by mnmin**

### CODE REVIEW

File: src/controllers/post.js [hunk 1]
```diff
@@ -52,9 +52,11 @@ export const getAll = async (req, res) => {
             }
           }
         }
-      }
+      },
+      comments: true
     }
   })
+  console.log('post', posts)
```
Comment (Auenc) [inline src/controllers/post.js:59]: Let's remove this `console.log`

File: src/controllers/post.js [hunk 2]
```diff
@@ -151,6 +153,41 @@ export const deletePost = async (req, res) => {
...
+  const { content } = req.body
+  const postId = Number(req.params.id)
+  const userId = req.user.id
+
+  const findPostById = await dbClient.post.findUnique({
+    where: {
+      id: postId
+    }
+  })
+  console.log('find post by id', findPostById)
```
Comment (Auenc) [inline src/controllers/post.js:166]: Let's remove this `console.log`

File: src/controllers/post.js [hunk 3]
```diff
@@ -151,6 +153,41 @@ export const deletePost = async (req, res) => {
...
+
+  if (!findPostById) {
+    return sendMessageResponse(res, 500, 'No post found')
```
Comment (Auenc) [inline src/controllers/post.js:169]: Let's change this error code from `500` to `404`

File: src/controllers/post.js [hunk 4]
```diff
@@ -151,6 +153,41 @@ export const deletePost = async (req, res) => {
...
+  }
+  try {
+    const createdComment = await dbClient.comment.create({
+      data: {
+        content,
+        userId,
+        postId
+      }
+    })
+    console.log('created Comment', createdComment)
```
Comment (Auenc) [inline src/controllers/post.js:183]: Let's remove this `console.log`

**Review by Auenc — CHANGES_REQUESTED** [review]

**Review by Noemi890 — APPROVED** [review]

**Pull request merged**